packages/apps/reputation-oracle/server/src/common/utils/cron.ts

```diff
@@ -0,0 +1,8 @@
+import { BACKOFF_INTERVAL_SECONDS } from '../constants';
+
+export function calculateBackoff(
```

Comment:
@dnechay Agree, since the function calculates an exponential backoff, a good name could be `backoff.ts`. If we find that we add more utility functions related to retries, we could consider grouping them into a larger `retry.ts` module.